src/Tizen.NUI.Components/Controls/Tab.cs

```diff
@@ -554,8 +554,8 @@ protected virtual void LayoutChild()
                     for (int i = 0; i < totalNum; i++)
                     {
                         preW = (itemList[i].NaturalSize2D != null ? itemList[i].NaturalSize2D.Width : 0);
-                        itemList[i].Position2D.X = w - preW - preX;
-                        itemList[i].Size2D.Width = preW;
+                        itemList[i].PositionX = w - preW - preX;
```

Comment:
I think this would be fine but just for double checking, could you add some TC or sample app to verify whether this works?
I mean set values with Position(xxx, xxx) and check if the PositionX and PositionY has same values and vice versa.

Reply:
Modify ViewSizeWidthPropertySetTest sample